crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -46,6 +49,10 @@ pub struct DWalletMPCManager {
 /// The possible results of verifying an incoming output for an MPC session.
 /// We need to differentiate between a duplicate & a malicious output, as the output can be sent twice by honest parties.
 pub enum OutputVerificationResult {
+    /// When working on a batch, e.g. signing on a batch of messages, we write the output to the chain only once - when the entire batch is ready.
+    ValidWithNewOutput(Vec<u8>),
```

Comment:
I think `ValidWithOutput` is a enough or maybe `ValidOutput` & `WaitingForOutputs`

Reply:
I prefer the current names